docs/server-side/v2/features/session.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: Session
+nav: 4.2
+---
+
+## Introduction
+
+Session is a single continuous call in a [room](room). To allow users to join a 100ms video conferencing session inside your app, you first need to create a `room`. A single room can have multiple sessions. You will need a [management token](../foundation/authentication-and-tokens) as a bearer token for your requests.
+
+## Get Session Data using API
+
+You can query session data using API.
+
+<GetRequest
+    title="Get Session API"
+    desp="This endpoint is used to get sessions. "
+    url="https://api.100ms.live/v2/sessions"
+/>
+
+<Tabs items={['Request', 'Response']} id="req" />
+
+<Tab id="req-0">
+
+```bash
+curl --location --request GET 'https://api.100ms.live/v2/sessions?url_param1=param1_value&url_params2=param2_value' -H 'Authorization: Bearer {{management_token}}'
```

Comment:
Make this multi line, helps in readability
See rtmp-streaming-and-browser-recording